runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -191,7 +196,13 @@ private FileDescriptor getDescriptor(String name) throws DescriptorValidationExc
      */
     private String getMessageName(Any message) {
         String typeUrl = message.getTypeUrl();
-        return typeUrl.substring(typeUrl.lastIndexOf('.') + 1);
+        String messageName = typeUrl.substring(typeUrl.lastIndexOf('.') + 1);
+        if (messageName.contains("/")) {
+            // In case the message lacks package name
+            // E.g. typeUrl: type.googleapis.com/TableName
+            messageName = messageName.substring(messageName.lastIndexOf("/") + 1);
```

Comment:
Did you print out the names of packages that contained a "/"? In my testing, all I could find were with "."

Reply:
Yes, inside RegistryTable there are some tables look like this `type.googleapis.com/TableName`

Reply:
Actually I just reviewed the RegistryTable in the setup, and there isn't this pattern. But I remember I did encounter an issue when I didn't handle this pattern in another setup I created using newer impactor build (which is no longer on Jarvis). Let's keep it as it is in case any such pattern appears.